test/integration/test_ibm_job_attributes.py

```diff
@@ -0,0 +1,325 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test IBMJob attributes."""
+
+import re
+import time
+import uuid
+from datetime import datetime, timedelta
+from unittest import mock, skip
+
+from dateutil import tz
+from qiskit.compiler import transpile
+from qiskit.providers.jobstatus import JobStatus, JOB_FINAL_STATES
+from qiskit import QuantumCircuit
+from qiskit.test.reference_circuits import ReferenceCircuits
+
+from qiskit_ibm_provider.api.clients.runtime import RuntimeClient
+from qiskit_ibm_provider.exceptions import (
+    IBMBackendValueError,
+)
+from qiskit_ibm_provider.job.exceptions import IBMJobFailureError
+
+from qiskit_ibm_runtime import IBMBackend, RuntimeJob
+from ..decorators import (
+    IntegrationTestDependencies,
+    integration_test_setup,
+)
+from ..ibm_test_case import IBMTestCase
+from ..utils import (
+    most_busy_backend,
+    cancel_job_safe,
+    submit_job_bad_shots,
+    submit_job_one_bad_instr,
+)
+
+
+class TestIBMJobAttributes(IBMTestCase):
+    """Test IBMJob instance attributes."""
+
+    sim_backend: IBMBackend
+    bell: QuantumCircuit
+    sim_job: RuntimeJob
+    last_week: datetime
+
+    @classmethod
+    @integration_test_setup()
+    def setUpClass(cls, dependencies: IntegrationTestDependencies) -> None:
+        """Initial class level setup."""
+        # pylint: disable=arguments-differ
+        super().setUpClass()
+        cls.dependencies = dependencies
+        cls.service = dependencies.service
+        cls.sim_backend = dependencies.service.backend(
+            "ibmq_qasm_simulator", instance=dependencies.instance
```

Comment:
This will also have to updated because the instance keyword is only for the quantum channel (they'll fail anyway right now because `circuit-runner` has not been enabled on the cloud side yet).

Reply:
Done.